src/DataStructures/Python/Tensor/TensorData.cpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <memory>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <sstream>
+#include <string>
+#include <utility>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/TensorData.hpp"
+#include "PythonBindings/BoundChecks.hpp"
+#include "Utilities/GetOutput.hpp"
+#include "Utilities/StdHelpers.hpp"
```

Comment:
Delete these includes: `memory`, `sstream`, `utility`, `BoundChecks`, `StdHelpers`

Add these includes: `vector`

Reply:
Ok removed these except StdHelpers because I need this for the str() and repr() functions in ExtentsAndTensorVolumeData.